### DISCUSSION TITLE
Realizzazione di un NVR opensource

### DISCUSSION THREAD

**Discussion by alessandroadorna** [indomus/forum#2396] [HUB: Home Assistant]
Ciao a tutti! Vi scrivo perchè vorrei un vostro consiglio visto la profonda competenza che vi contraddistingue. Lavoro come informatico per una grande azienda online che si occupa di sicurezza domestica. Al fine di realizzare un nvr "artigianale", nostro, più sicuro rispetto ai nvr e server cinesi che rivendiamo con telecamere comprese nel kit , cosa mi suggerite di fare? Realizzare un serverino low cost con una distro che permetta l'uso di home assistant e successivamente un applicativo adatto alla videosorveglianza? (camera.ui, ecc)
La necessità di un cliente basico è semplicemente quello di controllare 4-8 telecamere al max e ricevere le notifiche su smartphone. L'analisi delle immagini al fine di ridurre al minimo i falsi allarmi o per il riconoscimenti degli individui potrebbe essere un must ma non indispensabile.
Vi ringrazio per la vostra disponibilità e mi scuso per i pochi elementi indicati nel messaggio.

**Comment by sky7176** [comment]
Ciao,
non credo che HA sia il prodotto adatto per realizzare un NVR.

Ma onestamente sei veramente intenzionato a ricreare la ruota?

Installo impianti di video sorveglianza e la prima cosa che faccio è disabilitare il cloud su tutti gli NVR cinesi o meno.

Se proprio esiste l'esigenza di collegarsi per vedere le telecamera da remoto, una bella vpn e punti direttamente all'NVR.

Sono consapevole che per l'utente medio potrebbe suonare strano, ma credimi i casini con la violazione della privacy sono molto ma molto più seri.

Giuseppe

**Reply by alessandroadorna** [reply]
Ciao Giuseppe, grazie per il tuo commento. La necessità di una maggiore sicurezza è più un cavillo mio personale che una richiesta della clientela. Tieni conto che la maggior parte chiede assistenza per scaricare un’app e registrarsi, figurati se avvertono la necessità di una maggiore protezione. 
La mia priorità è:
Possibilità di ricevere notifiche
Bassa immunità ai falsi allarmi
Investimento hw medio/basso per l’azienda.
Vorrei valutare HA perché alcuni progetti sembrano soddisfare questi requisiti. Se poi l’esperienza d’uso è accettabile o meno lo scoprirò provandolo 🥴

**Reply by KC-inDomus** [reply]
ma questa cosa la devi fare per usarla internamente o per venderla?

**Reply by alessandroadorna** [reply]
venderla

**Reply by KC-inDomus** [reply]
beh, auguri. home assistant non lo puoi usare per rivenderlo salvo violare la licenza open source, così come altri oggetti eventualmente disponibili

**Comment by alessandroadorna** [comment]
Ah, no, ok..allora come non detto. Grazie per il vostro supporto.
